x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/index.tsx

```diff
@@ -17,7 +17,7 @@ import { LicensePrompt } from '../../../shared/LicensePrompt';
 import { useLicense } from '../../../../hooks/useLicense';
 import { APIReturnType } from '../../../../services/rest/createCallApmApi';
 
-export type AnomalyDetectionApiResponse = APIReturnType<'GET /api/apm/settings/anomaly-detection'>;
+export type AnomalyDetectionApiResponse = APIReturnType<'GET /api/apm/settings/anomaly-detection/jobs'>;
```

Comment:
Wouldn't be better if the API returned the type instead of having it defined in multiple files?

Reply:
Defining this on the server will bring back the eslint import error. We could extract the hook into a separate file (it's also duplicated at the moment), ad co-locate the type but that will cause some other problems wrt conditional fetching.